src/components/forms/pool_actions/MigrateForm/constants.ts

```diff
@@ -87,6 +87,8 @@ const POLYGON_POOL_MIGRATIONS: PoolMigrationInfo[] = [
   },
 ];
 
+export const migratablePoolsInvestExceptions = [POOLS.IdsMap.mai4?.mai4];
```

Comment:
This is a difficult one to name, because it's not just about the investment but also about not showing specific warnings. I'd suggest this is a class of migration that we could call 'soft'. It's soft in that we still present users with the option to migrate liquidity and keep the warning on the pool page, but we don't highlight this to the user in the pools list or block them from investing in the pool. So maybe we could change the name of this variable and a comment to describe this, something like:
```suggestion
/**
 * Checks if a pool is classified as a soft migration, a soft migration meaning that you can still invest in the 
 *  current pool and we don't highlight a forced migration for current LPs.
 */
export function isSoftMigratablePool(id: string) {
...
}
```

Reply:
hm, yeah, think it is better. thanks! 
fixed